Twenty403/src/main/java/org/firstinspires/ftc/twenty403/controls/ControlOperator.java

```diff
@@ -60,40 +50,40 @@ private void AssignNamedControllerButton() {
     }
 
     public void bindClawControls() {
-        clawOpenButton.whenPressed(new ClawOpenCommand(robot.clawSubsystem));
-        clawCloseButton.whenReleased(new ClawCloseCommand(robot.clawSubsystem));
+        clawOpenButton.whenPressed(robot.clawSubsystem.openCommand);
```

Comment:
Why now we have three ways to do this? through a dot object, a colon function, and simple required command? What's the difference?

Reply:
My primary goal is to simplify the work of manipulating the robot through code. The vast majority of commands are 25-30 lines files, that have precisely 2 lines of real code (`addRequirements(subsys)` in the constructor, and then `subsys.doTheThing()` in the `execute` method) There are a few different ways we could structure things to minimize the amount of copy-pasta. I think the library, as it's currently authored, enables lots of different ways to accomplish what we're talking about. Perhaps removing all the CommandButton overloads, and instead encouraging a static class of commands for each subsystem that are consistent in their usage would be easier to follow? I'll try a minor change and modify this PR with a different possibility (after I've diagnosed a null pointer exception, and maybe added some better logging to help tracking them down easier in the future)